backend/src/data/models/appointment.ts

```diff
@@ -0,0 +1,34 @@
+import { Sequelize, DataTypes, ModelCtor, Model } from 'sequelize';
+
+enum Type {
+  ONLINE = 'online',
+  OFFLINE = 'offline'
+}
+
+interface AppointmentInstance extends Model {
+  date: Date;
+  type: Type;
+  cost: number;
+  subject: string;
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+export default (orm: Sequelize): ModelCtor<AppointmentInstance> => {
+  const Appointment = orm.define<AppointmentInstance>('appointment', {
```

Comment:
```suggestion
  const Appointment = orm.define<AppointmentInstance>(ModelName.APPOINTMENT, {
```
It would be better to have the enum for model name